src/app/tasks/tasks.component.html

```diff
@@ -22,10 +22,20 @@
         imgUrlPrefix + task.assignee.avatar :
         'assets/image.png'
     ">
-    <button mat-icon-button [matMenuTriggerFor]="assigneeMenu" (click)="openAssigneeMenu($event)"><mat-icon>add</mat-icon></button>
-    <mat-menu #assigneeMenu="matMenu">
-      <button *ngIf="task.assignee" (click)="addAssignee(task)" mat-menu-item><mat-icon>clear</mat-icon> <ng-container i18n>Remove</ng-container> {{task.assignee.name}}</button>
-      <button *ngFor="let assignee of (assignees | filterAssignee: task.assignee)" mat-menu-item (click)="addAssignee(task, assignee)">{{assignee.name}}</button>
-    </mat-menu>
-  </button>
+    <button class="top-right-icon" mat-icon-button [matMenuTriggerFor]="resourceMenu">
+    <mat-icon>more_vert</mat-icon>
+    </button>
+  <mat-menu #resourceMenu="matMenu">
+    <button mat-menu-item [matMenuTriggerFor]="assigneeMenu" (click)="openAssigneeMenu($event)">
+      <mat-icon>add</mat-icon>
+      <span>Add</span>
+      <mat-menu #assigneeMenu="matMenu">
+        <button *ngIf="task.assignee" (click)="addAssignee(task)" mat-menu-item><mat-icon>clear</mat-icon> <ng-container i18n>Remove</ng-container> {{task.assignee.name}}</button>
+        <button *ngFor="let assignee of (assignees | filterAssignee: task.assignee)" mat-menu-item (click)="addAssignee(task, assignee)">{{assignee.name}}</button>
+      </mat-menu>
+    </button>
+    <button mat-menu-item (click)="addTask(); $event.stopPropagation()" i18n><mat-icon>edit</mat-icon> Edit</button>
```

Comment:
Let's put these buttons in a separate menu to the right of the assign member menu.  I don't think we should hide the menu for assigning a member.

Reply:
ok thank you paul.. please check the final work done @paulbert